packages/@okta/vuepress-site/code/react/okta_react/index.md

```diff
@@ -39,31 +39,89 @@ A simple way to add authentication to a React app is using the [Okta Auth JS](/c
 npm install @okta/okta-auth-js --save
 ```
 
-We'll also need `@okta/okta-react` and `react-router-dom` to manage our routes:
+We'll also need `@okta/okta-react` and `react-router-dom` to manage our routes (`@okta/okta-react` can be used to support other router libraries, but `react-router-dom` has pre-existing support).
 ```bash
 npm install @okta/okta-react react-router-dom --save
```

Comment:
Should we split this script to make `react-router-dom` optional, also mention potential npm warning?

Reply:
This is a get-started script, so we're showing them the fastest way to get started.  

The optional-ness and warning-if-omitted are covered by okta-react docs, this is more showing them that the library is worthwhile.  (it's also due to be replaced by our guides, but that's a slower process).